### PR TITLE
Don't publish from PRs

### PR DESCRIPTION
**Pull request by brianjmurrell**

### CODE REVIEW

File: vars/publishToRepository.groovy [hunk 1]
```diff
@@ -35,6 +35,12 @@ def call(Map config = [:]) {
         println('No REPOSITORY_URL environment variable configured.')
         return
     }
+
+    // Don't publish from PRs
+    if (env.CHANGE_ID) {
```
Comment (JohnMalmberg) [inline vars/publishToRepository.groovy]: There are some cases where we want to publish from a PR, usually into a test repository, such as the self test for this module.

Comment (brianjmurrell) [reply vars/publishToRepository.groovy]: Hrm.  How did the tests pass then?

Comment (JohnMalmberg) [reply vars/publishToRepository.groovy]: Tests are actually only run / graded if the test parameter is passed to the publishToRepositorySystem if that routine exists and the repository is not an APT repository.

Comment (brianjmurrell) [reply vars/publishToRepository.groovy]: But the tests were run for this PR.

**Pull request merged**